x-pack/plugins/security_solution/server/lib/detection_engine/rules/prepackaged_rules/persistence_startup_folder_scripts.json

```diff
@@ -0,0 +1,43 @@
+{
+  "author": [
+    "Elastic"
+  ],
+  "description": "Identifies script engines creating files in the startup folder, or the creation of script files in the startup folder.",
+  "index": [
+    "winlogbeat-*",
+    "logs-endpoint.events.*"
+  ],
+  "language": "eql",
+  "license": "Elastic License",
+  "name": "Persistent Scripts in the Startup Directory",
+  "query": "file where event.type != \"deletion\" and user.domain != \"NT AUTHORITY\"\n  and (\n    // detect shortcuts created by wscript.exe or cscript.exe\n      file.path : \"C:\\\\*\\\\Programs\\\\Startup\\\\*.lnk\" and\n      process.name : (\"wscript.exe\", \"cscript.exe\")\n      ) or\n     // detect vbs or js files created by any process\n     file.path : (\"C:\\\\*\\\\Programs\\\\Startup\\\\*.vbs\", \n                  \"C:\\\\*\\\\Programs\\\\Startup\\\\*.vbe\", \n                  \"C:\\\\*\\\\Programs\\\\Startup\\\\*.wsh\", \n                  \"C:\\\\*\\\\Programs\\\\Startup\\\\*.wsf\", \n                  \"C:\\\\*\\\\Programs\\\\Startup\\\\*.js\")\n",
```

Comment:
For this one though, is this a template?
<p align="center">
  <img width="500" src="https://user-images.githubusercontent.com/2946766/101856459-946c8980-3b22-11eb-8e64-be239f54f6c8.png" />
</p>

Reply:
just more annotation, though I see this one used line comments rather than block, so I will make an issue to review and remove all line comments 👍

Reply:
https://github.com/elastic/detection-rules/issues/708